### ISSUE TITLE
Display loss?

### ISSUE THREAD

**Issue by CraigWang1** [google/automl#53]
Is there a way to display loss when training on GPUs?

Currently, all I see are the steps:
```
I0326 05:18:42.526468 139907207948160 tpu_estimator.py:2308] examples/sec: 14.628
INFO:tensorflow:global_step/sec: 0.922679
I0326 05:18:43.609837 139907207948160 tpu_estimator.py:2307] global_step/sec: 0.922679
INFO:tensorflow:examples/sec: 14.7629
I0326 05:18:43.610084 139907207948160 tpu_estimator.py:2308] examples/sec: 14.7629
INFO:tensorflow:global_step/sec: 0.96926
```

**Comment by mingxingtan** [comment]
How about adding --logtostderr when running "python main.py ..."?  (You should also be able to view the loss in tensorboard)

**Comment by mingxingtan** [comment]
Since many people want to see loss in the output, I have changed the log level to LOGGING. Now you should be able to see the loss.

**Comment by cyx6666** [comment]
> Since many people want to see loss in the output, I have changed the log level to LOGGING. Now you should be able to see the loss.

Is this problem already be solved?
I still don't see loss value
`0401 05:59:02.752293 140096867051392 tpu_estimator.py:2351] examples/sec: 1.14218
INFO:tensorflow:global_step/sec: 1.15993
I0401 05:59:03.614092 140096867051392 tpu_estimator.py:2350] global_step/sec: 1.15993
INFO:tensorflow:examples/sec: 1.15993
I0401 05:59:03.614437 140096867051392 tpu_estimator.py:2351] examples/sec: 1.15993
INFO:tensorflow:global_step/sec: 1.15224
I0401 05:59:04.481988 140096867051392 tpu_estimator.py:2350] global_step/sec: 1.15224
INFO:tensorflow:examples/sec: 1.15224
I0401 05:59:04.482370 140096867051392 tpu_estimator.py:2351] examples/sec: 1.15224
INFO:tensorflow:global_step/sec: 1.13129
I0401 05:59:05.365908 140096867051392 tpu_estimator.py:2350] global_step/sec: 1.13129
INFO:tensorflow:examples/sec: 1.13129
I0401 05:59:05.366242 140096867051392 tpu_estimator.py:2351] examples/sec: 1.13129
INFO:tensorflow:global_step/sec: 1.1429
I0401 05:59:06.240884 140096867051392 tpu_estimator.py:2350] global_step/sec: 1.1429
INFO:tensorflow:examples/sec: 1.1429
I0401 05:59:06.241212 140096867051392 tpu_estimator.py:2351] examples/sec: 1.1429
INFO:tensorflow:global_step/sec: 1.12954`

**Comment by Li-by** [comment]
@cyx6666 Is there a way to display loss when training on GPUs?

**Comment by CraigWang1** [comment]
Hi, you can use tensorboard to look at loss.

In Google Colab it would like like:
```
%load_ext tensorboard
%tensorboard --logdir </path/to/model_dir>
```
And then it will generate loss and mAP graphs for you in real time.
